gufe/settings/models.py

```diff
@@ -6,40 +6,22 @@
 
 import abc
 import pprint
-from typing import Optional, Union
+from typing import Literal
 
 from openff.units import unit
+from pydantic import AfterValidator, ConfigDict, Field, PositiveFloat, PrivateAttr, field_validator, validator
 
-from gufe.vendor.openff.models.models import DefaultModel
-from gufe.vendor.openff.models.types import FloatQuantity
+from gufe.vendor.openff.interchange._annotations import _DistanceQuantity, _PressureQuantity, _TemperatureQuantity
```

Comment:
Just a comment for "before we consider merging", I would very much like it if these were not private types.

Reply:
these are removed now since we don't do non-converted types here. the only private methods im keeping are because theyre in vendored code and i want to keep those names identical for code provenance.